egs/fisher_swbd/s5/RESULTS

```diff
@@ -42,7 +42,51 @@ for x in exp/nnet2_online/nnet_ms_a_online/decode_eval2000*_fg; do grep Sum  $x/
 %WER 12.3 | 1831 21395 | 89.2 7.2 3.5 1.5 12.3 50.8 | exp/nnet2_online/nnet_ms_a_online/decode_eval2000_utt_fsh_sw1_fg/score_13/eval2000.ctm.swbd.filt.sys
 %WER 11.8 | 1831 21395 | 89.6 7.2 3.2 1.4 11.8 49.0 | exp/nnet2_online/nnet_ms_a_online/decode_eval2000_utt_offline_fsh_sw1_fg/score_11/eval2000.ctm.swbd.filt.sys
 
+# nnet3 result on eval2000
+# BLSTM ran for about 760 hours, command:
+# local/nnet3/run_lstm.sh --affix bidirectional --lstm-delay " [-1,1] [-2,2] [-3,3] " --label-delay 0 \
+#                         --cell-dim 1024 --recurrent-projection-dim 128 --non-recurrent-projection-dim 128 \
+#                         --chunk-left-context 40 --chunk-right-context 40 \
+#                         --extra-left-context 50 --extra-right-context 50
+# use tri-gram
+for x in exp/nnet3/*/decode_eval2000*tg; do grep Sum  $x/score_*/*.ctm.filt.sys | utils/best_wer.sh ; done
+%WER 15.8 | 4459 42989 | 86.1 9.7 4.1 1.9 15.8 52.6 | exp/nnet3/tdnn_sp/decode_eval2000_fsh_sw1_tg/score_10_0.0/eval2000_hires.ctm.filt.sys
+%WER 14.8 | 4459 42989 | 86.6 9.2 4.3 1.4 14.8 54.3 | exp/nnet3/lstm_bidirectional_sp/decode_eval2000_fsh_sw1_tg/score_10_0.0/eval2000_hires.ctm.filt.sys
+# rescore with four-gram
+for x in exp/nnet3/*/decode_eval2000*fg; do grep Sum  $x/score_*/*.ctm.filt.sys | utils/best_wer.sh ; done
+%WER 15.4 | 4459 42989 | 86.4 9.5 4.0 1.8 15.4 51.6 | exp/nnet3/tdnn_sp/decode_eval2000_fsh_sw1_fg/score_10_0.0/eval2000_hires.ctm.filt.sys
+%WER 14.5 | 4459 42989 | 87.0 9.0 4.0 1.5 14.5 53.7 | exp/nnet3/lstm_bidirectional_sp/decode_eval2000_fsh_sw1_fg/score_8_0.0/eval2000_hires.ctm.filt.sys
 
+# nnet3 result on eval2000 for swbd subset
+# use tri-gram
+for x in exp/nnet3/*/decode_eval2000*tg; do grep Sum  $x/score_*/*.ctm.swbd.filt.sys | utils/best_wer.sh ; done
+%WER 11.6 | 1831 21395 | 89.7 7.3 3.0 1.3 11.6 47.7 | exp/nnet3/tdnn_sp/decode_eval2000_fsh_sw1_tg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys
+%WER 10.7 | 1831 21395 | 90.3 6.7 3.0 1.0 10.7 45.9 | exp/nnet3/lstm_bidirectional_sp/decode_eval2000_fsh_sw1_tg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys
+# rescore with four-gram
+for x in exp/nnet3/*/decode_eval2000*fg; do grep Sum  $x/score_*/*.ctm.swbd.filt.sys | utils/best_wer.sh ; done
+%WER 11.1 | 1831 21395 | 90.2 7.0 2.8 1.3 11.1 46.2 | exp/nnet3/tdnn_sp/decode_eval2000_fsh_sw1_fg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys
+%WER 10.4 | 1831 21395 | 90.6 6.5 2.9 1.0 10.4 45.3 | exp/nnet3/lstm_bidirectional_sp/decode_eval2000_fsh_sw1_fg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys
+
+# chain result on eval2000
+# BLSTM ran for about 380 hours
```

Comment:
I'll fix it.

在 2016/4/14 10:48, Daniel Povey 写道:
>
> In egs/fisher_swbd/s5/RESULTS 
> <https://github.com/kaldi-asr/kaldi/pull/657#discussion_r59657272>:
>
> >
> > +# nnet3 result on eval2000 for swbd subset
> > +# use tri-gram
> > +for x in exp/nnet3/*/decode_eval2000*tg; do grep Sum  $x/score_*/*.ctm.swbd.filt.sys | utils/best_wer.sh ; done
> > +%WER 11.6 | 1831 21395 | 89.7 7.3 3.0 1.3 11.6 47.7 | exp/nnet3/tdnn_sp/decode_eval2000_fsh_sw1_tg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys
> > +%WER 10.7 | 1831 21395 | 90.3 6.7 3.0 1.0 10.7 45.9 | exp/nnet3/lstm_bidirectional_sp/decode_eval2000_fsh_sw1_tg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys
> > +# rescore with four-gram
> > +for x in exp/nnet3/*/decode_eval2000*fg; do grep Sum  $x/score_*/*.ctm.swbd.filt.sys | utils/best_wer.sh ; done
> > +%WER 11.1 | 1831 21395 | 90.2 7.0 2.8 1.3 11.1 46.2 | exp/nnet3/tdnn_sp/decode_eval2000_fsh_sw1_fg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys
> > +%WER 10.4 | 1831 21395 | 90.6 6.5 2.9 1.0 10.4 45.3 | exp/nnet3/lstm_bidirectional_sp/decode_eval2000_fsh_sw1_fg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys
> > +
> > +# chain result on eval2000
> > +# BLSTM ran for about 380 hours
> That is a bug in the recipe- probably a copy-paste error. Can you 
> please either make a PR for it or create an issue?
> … <#>
> On Wed, Apr 13, 2016 at 7:46 PM, Xingyu Na <notifications@github.com> 
> wrote: In egs/fisher_swbd/s5/RESULTS <#657 (comment) 
> <https://github.com/kaldi-asr/kaldi/pull/657#discussion_r59657141>>: > 
> > +# nnet3 result on eval2000 for swbd subset > +# use tri-gram > +for 
> x in exp/nnet3/*/decode_eval2000*tg; do grep Sum 
> $x/score_*/*.ctm.swbd.filt.sys | utils/best_wer.sh ; done > +%WER 11.6 
> | 1831 21395 | 89.7 7.3 3.0 1.3 11.6 47.7 | 
> exp/nnet3/tdnn_sp/decode_eval2000_fsh_sw1_tg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys 
> > +%WER 10.7 | 1831 21395 | 90.3 6.7 3.0 1.0 10.7 45.9 | 
> exp/nnet3/lstm_bidirectional_sp/decode_eval2000_fsh_sw1_tg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys 
> > +# rescore with four-gram > +for x in 
> exp/nnet3/*/decode_eval2000*fg; do grep Sum 
> $x/score_*/*.ctm.swbd.filt.sys | utils/best_wer.sh ; done > +%WER 11.1 
> | 1831 21395 | 90.2 7.0 2.8 1.3 11.1 46.2 | 
> exp/nnet3/tdnn_sp/decode_eval2000_fsh_sw1_fg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys 
> > +%WER 10.4 | 1831 21395 | 90.6 6.5 2.9 1.0 10.4 45.3 | 
> exp/nnet3/lstm_bidirectional_sp/decode_eval2000_fsh_sw1_fg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys 
> > + > +# chain result on eval2000 > +# BLSTM ran for about 380 hours 
> Sorry I mean the the decoding dir with the name 'dev' is actually 
> eval2000, e.g. L150 of run.sh. I don't know why. 在 2016/4/14 10:35, 
> Daniel Povey 写道: … <#m_412739808674561755_> In 
> egs/fisher_swbd/s5/RESULTS <#657 
> <https://github.com/kaldi-asr/kaldi/pull/657> (comment) <#657 
> (comment) 
> <https://github.com/kaldi-asr/kaldi/pull/657#discussion_r59656643>>>: 
> > > +# nnet3 result on eval2000 for swbd subset > +# use tri-gram > 
> +for x in exp/nnet3/*/decode_eval2000*tg; do grep Sum 
> $x/score_*/*.ctm.swbd.filt.sys | utils/best_wer.sh ; done > +%WER 11.6 
> | 1831 21395 | 89.7 7.3 3.0 1.3 11.6 47.7 | 
> exp/nnet3/tdnn_sp/decode_eval2000_fsh_sw1_tg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys 
> > +%WER 10.7 | 1831 21395 | 90.3 6.7 3.0 1.0 10.7 45.9 | 
> exp/nnet3/lstm_bidirectional_sp/decode_eval2000_fsh_sw1_tg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys 
> > +# rescore with four-gram > +for x in 
> exp/nnet3/*/decode_eval2000*fg; do grep Sum 
> $x/score_*/*.ctm.swbd.filt.sys | utils/best_wer.sh ; done > +%WER 11.1 
> | 1831 21395 | 90.2 7.0 2.8 1.3 11.1 46.2 | 
> exp/nnet3/tdnn_sp/decode_eval2000_fsh_sw1_fg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys 
> > +%WER 10.4 | 1831 21395 | 90.6 6.5 2.9 1.0 10.4 45.3 | 
> exp/nnet3/lstm_bidirectional_sp/decode_eval2000_fsh_sw1_fg/score_10_0.0/eval2000_hires.ctm.swbd.filt.sys 
> > + > +# chain result on eval2000 > +# BLSTM ran for about 380 hours 
> The so called 'train_dev' subset in fisher_swbd is eval2000. I'm not 
> sure what you mean by this statement. I don't see any 'train_dev' 
> subset defined in the run.sh of fisher_swbd. In the swbd setup it's 
> just a subset of the training data. It doesn't make sense to use it as 
> an alias for a NIST-defined test set. — You are receiving this because 
> you were mentioned. Reply to this email directly or view it on GitHub 
> < 
> https://github.com/kaldi-asr/kaldi/pull/657/files/fc4140efa4727c4ad11cf689ae3faa26a9d0ffa0#r59656643> 
> — You are receiving this because you were mentioned. Reply to this 
> email directly or view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/657/files/fc4140efa4727c4ad11cf689ae3faa26a9d0ffa0#r59657141> 
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/657/files/fc4140efa4727c4ad11cf689ae3faa26a9d0ffa0#r59657272>
>